README.md

```diff
@@ -2,3 +2,5 @@
 Learn github flow.
 Forking workflow.
 update 1
+update 2
+update 3
```

Comment:
I dont like this idea

Reply:
I did as you suggested. Please merge if everything is ok